docs/src/main/tut/dsl.md

```diff
@@ -214,25 +214,26 @@ import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 ```
 
-You can seamlessly respond with a `Future` of any type that has an
-`EntityEncoder`.
+You can respond with a `Future` of any type that has an
+`EntityEncoder` by lifting it into IO. Note: `Future` is a side effecting and
```

Comment:
Pedantry: future is pure.  The problems are that it's strict (so creating a Future with an effect is an effect) and that mapping over it is an effect.

Reply:
`s/effect/side effect`.
You are correct, but how many people do you know that use Future with pure code inside? (I know of 0, they introduce Future to deal with side effects)

Reply:
I know of 0, but I know of non-zero who point this out every time someone calls `Future` impure on Twitter. :)

Reply:
But anyway you're right, this could probably we reworded like:
```
Note: unlike IO, wrapping a side effect in Future does not suspend it, and the resulting expression would still be side effectful, unless we wrap it in IO
```